app/models/upload.rb

```diff
@@ -116,9 +116,14 @@ def self.fetching_for?(csv_type)
     Upload.where(ok: false, completed_at: nil, csv_type: csv_type).any?
   end
 
+  # def self.valid_col_seps
+  #   valid_col_seps = Settings.csv_upload.column_separators.map { |cs| "\"#{cs}\"" }.join(' and ')
+  #   "Valid column separators are: #{valid_col_seps}."
+  # end
+
   def self.valid_col_seps
```

Comment:
how does changing this method affect the usage in the method `Upload.set_col_sep` down below in this file?